bindings/cs/rl.net.native/rl.net.live_model.h

```diff
@@ -2,11 +2,14 @@
 
 #include "rl.net.native.h"
 
-typedef void (*managed_callback_t)(const reinforcement_learning::api_status&);
+typedef void (*background_error_callback_t)(const reinforcement_learning::api_status&);
+typedef void (*trace_logger_t)(int log_level, const char* msg);
```

Comment:
Callbacks should always include an user-supplied pointer, otherwise it makes any statefull implementations incredibly complicated.